apps/sms/index.html

```diff
@@ -80,6 +80,13 @@
     <script defer src="shared/js/fb/fb_data_reader.js"></script>
```

Comment:
I have no idea about this warning :/ But I can try to find which script triggered this warning.